sway-lib-std/src/assert.sw

```diff
@@ -1,25 +1,14 @@
 library assert;
 
-use ::logging::log;
 use ::revert::revert;
 
-const FAILED_REQUIRE_SIGNAL = 42;
-
-/// Assert that a value is true
-pub fn assert(a: bool) {
-    if !a {
+/// Asserts that the given `condition` will always be `true` during runtime.
+/// To check for conditions that may not be `true`, use `std::revert` instead.
```

Comment:
@AlicanC Was this meant to read:
 "/// To check for conditions that may not be `true`, use `std::revert::require` instead." ?

Reply:
Yup, fixed.